modules/ipi-install-modifying-install-config-for-dual-stack-network.adoc

```diff
@@ -41,6 +41,23 @@ platform:
       - <wildcard_ipv6>
 ----
 
+[NOTE]
+====
+For dual-stack clusters that use Stateless Address AutoConfiguration (SLAAC), you must use the same value for the `addr-gen-mode` connection setting in both the installation ramdisk and the node instance. 
+
+You can set the `addr-gen-mode` connection settings in the node instance by using the nmcli command:
+
+[source,terminal]
+----
+$ nmcli connection modify <connection_name> addr-gen-mode <value> <1>
```

Comment:
I think we want to do this via a machine-config so it applies on first boot. If we use a cli command we'll get an incorrect IP when the node comes up and that will cause issues with the deployment. We need to create a file with the contents discussed in [this commit message](https://gitlab.freedesktop.org/NetworkManager/NetworkManager/-/commit/e6a33c04ebe1ac84e31628911e25bdfd7534dd3c), preferably by passing a machine-config as an install-time manifest so it's there from day 1.

The one downside of using the method is that I believe it will only work in 4.13 and up. 4.12 and older don't have a new enough NetworkManager to support this global option.